Strassen Matrix Multiplication/README.markdown

```diff
@@ -0,0 +1 @@
+## TODO: Examples & Explanation
```

Comment:
Looking forward to this! Linear Algebra seems to be a prerequisite for this article, so it'll be good to mention they should have some background on working with Matrices, and provide a link to where you would recommend them to get some basics down before looking into this.

You're also making use of your own helper protocols and defining your own overloaded operators, so make sure you make clear references to avoid confusion for the readers when they follow along.